### PR TITLE
Better Multi-CI UI

### PR DESCRIPTION
**Pull request by jeromegn**
Changes:

- Bigger popover with details of all the checks
- Removed black tooltip for grouped status

### CODE REVIEW

**Comment by gmalette** [comment]
Can you provide a screenshot please?

**Comment by jeromegn** [comment]
@gmalette Sure!

Success:

image

Error & Failure:

image

Pending:

image

It's not perfect for the error and failure states (well, neither for the others to be honest), but it's a start. We'd probably need "bolder" icons for the smaller versions. They're a bit difficult to see.

**Comment by gmalette** [comment]
That's awesome!

**Comment by byroot** [comment]
Yep, definitely great! Just tell us when you think it's good and we'll merge!

**Comment by byroot** [comment]
As discussed outside of this thread, this is still worthy of a few minor improvements like making the icons bolder and adjusting colors. But I's already a huge improvement over #373 so I'll merge it in and continue work there.

**Comment by byroot** [comment]
Thanks a lot @jeromegn !

File: app/views/statuses/_status.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<a class="status status--<%= status.state %> <%= :disabled unless status.target_url.present? %>" data-tooltip="<%= status.description.presence || status.state.capitalize %>" href="<%= status.target_url%>" target="_blank">
+<a class="status status--<%= status.state %> <%= :disabled unless status.target_url.present? %>" <% unless status.group? %>data-tooltip="<%= status.description.presence || status.state.capitalize %>"<% end %> href="<%= status.target_url%>" target="_blank">
```
Comment (rafaelfranca) [inline app/views/statuses/_status.html.erb]: There is a lot of logic on this line. Maybe it is worth to extract it.

**Pull request merged**